src/commands/core/plugins/add.ts

```
@@ -1,19 +1,18 @@
 import { Args, Flags } from '@oclif/core'
 
-import { BaseCommand } from '../../../BaseCommand.js'
-import { Plugin, PluginData } from '../../../lib/types/pluginData.js'
-import { PluginFilterType, pluginSelector } from '../../../prompts/pluginSelector.js'
-import pluginConfigurator from '../../../prompts/pluginInquirer.js'
-import fs from 'fs'
-import { addCollectionPlugin, AddCollectionPluginArgs, AddCollectionPluginArgsPlugin, addPlugin, AddPluginArgs, AddPluginArgsPlugin } from '@metaplex-foundation/mpl-core'
+import { addCollectionPlugin, AddCollectionPluginArgsPlugin, addPlugin, AddPluginArgsPlugin } from '@metaplex-foundation/mpl-core'
 import { publicKey } from '@metaplex-foundation/umi'
-import umiSendAndConfirmTransaction from '../../../lib/umi/sendAndConfirm.js'
+import fs from 'fs'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)